core/settings/qPref.h

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: GPL-2.0
+#ifndef QPREF_H
+#define QPREF_H
+
+#include <QObject>
+#include "core/pref.h"
+
+class qPref : public QObject {
+	Q_OBJECT
+
+public:
+	qPref(QObject *parent = NULL) : QObject(parent) {};
+	~qPref() {};
+	static qPref *instance();
+
+	// Load/Sync local settings (disk) and struct preference
+	void loadSync(bool doSync);
```

Comment:
What is the advantage of this loadSync vs just loading as needed (or constructor) and just setting on setter?

I only see a centralized sync making sense if setters are not used (like in the existing code), but this is not the case I see on #1423 where qPref is used.

Reply:
well at least we need a centralized read. And when savi g it would be good to sync as well, as many change struct preferences directly (no set call)